eclipse-collections-api/src/main/java/org/eclipse/collections/api/RichIterable.java

```diff
@@ -104,6 +104,17 @@ default boolean notEmpty()
         return !this.isEmpty();
     }
 
+    /**
+     * Returns any element of an iterable.
+     *
+     * @return an element of an iterable.
+     * @since 10.0
+     */
+    default T getAny()
+    {
+        return this.getFirst();
```

Comment:
I think @motlin was suggesting to delete to `this.getFirst()` only for OrderedIterable. For other types, it should be implemented.
@motlin correct me if I am wrong.

Reply:
I understood the below comment to be as per my changes. But please let me know if ive misunderstood something here @motlin  

Motlin:
The fastest way to add getAny is as a default method on RichIterable that
delegates to getFirst but isn’t deprecated.

Reply:
I think this is fine. IF we ever wanted to remove getFirst from RichIterable, but leave it on OrderedIterable, then we'd have to push this implementation down and implement getAny like getFirst in places like sets. But we probably won't remove methods.

Reply:
Thanks @motlin , any other changes required on this PR?